pkg/cli/admin/release/extract_tools.go

```diff
@@ -674,6 +686,7 @@ func copyAndReplaceReleaseInfo(w io.Writer, r io.Reader, bufferSize int, release
 		}
 		if err != nil {
 			if err == io.EOF || err == io.ErrUnexpectedEOF {
+				klog.V(2).Infof("GOT HERE")
```

Comment:
oops, will remove but want to see it run thru CI/get a release image from this first.